### DISCUSSION TITLE
package up cypress and custom commands as a private npm package for organisation

### DISCUSSION THREAD

**Discussion by country-bumpkin-software** [cypress-io/cypress#20212] [General]
So where do I start. Im looking to create an e2e framework for my organisation that lots of different teams will be using. So I thought maybe the best thing to do would be to somehow create a repo with cypress in it and create all of my custom commands that the teams would probably use and somehow publish that to my organisations NPM?  

So what is the best way to go about this? Is it to just to create the custom command in the repo and then the team downloads the cypress normally and then download the npm packages with my custom commands, or is it normal to bundle everything together.  I haven't done this before so some docs or instructions and best practises would be greatly appreciated.

**Comment by NithinReddyVertex** [comment]
@country-bumpkin-software Hey there. I was thinking about this use case for my org as well. What did you end up doing?
